tests/test_sync.py

```diff
@@ -458,6 +459,17 @@ def test_sync_ask_accepted(check_call, confirm, from_line, dry_run):
 
     confirm.assert_called_once_with("Would you like to proceed with these changes?")
```

Comment:
I wonder could it be refactored to check the message in `std{err,out}` instead?

Reply:
Good idea and thanks for the review. I have implemented this in the latest revision.